src/api/doctors.api.js

```diff
@@ -0,0 +1,4 @@
+import { instance } from './instance.api';
+
+export const allAppointmentsForDoctor = () => instance
+  .get('/appointments/doctor/me?offset=0&limit=20');
```

Comment:
offset=0&limit=20, limit and offset should be passed in arguments

Reply:
ready